specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -156,6 +156,54 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DataFactory/locations/{locationId}/getFeatureValue": {
```

Comment:
What is the usecase for this API? Is it doing something similar to Azure Feature Exposure Control (AFEC)?

Reply:
@ravbhatnagar ADF doesn't use Azure Feature Exposure Control, instead we have our own implemented in our backend which controls some throttling limits and enabled disabled features for the customer, and this API is allowing the user to query for a specific value.